src/DssSpell.t.sol

```diff
@@ -1054,4 +1246,30 @@ contract DssSpellTest is DSTest, DSMath {
         address UNIUSD_MED  = 0x52f761908cC27B4D77AD7A329463cf08baf62153;
         assertEq(MedianAbstract(UNIUSD_MED).bud(SET_UNI), 1);
     }
+
+    function testOsmWhitelist() public {
+        vote();
+        scheduleWaitAndCast();
+        assertTrue(spell.done());
+
+        // Verify that B Protocol has been whitelisted on the ETHUSD Medianizer and OSM
+        assertEq(OsmAbstract(0x81FE72B5A8d1A857d176C3E7d5Bd2679A9B85763).bud(0xDF3CDd10e646e4155723a3bC5b1191741DD90333), 1);
+    }
+
+    function testPayouts() public {
+        vote();
+        scheduleWaitAndCast();
+        assertTrue(spell.done());
+
+        // Verify Vault Compensation Working Group Payment have been sent out properly
+        assertTrue(dai.balanceOf(0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0) >= 6300 ether);
```

Comment:
It would be better to catch the balances in the beginning and check `actual_balance = old_balance + new value`